motionplan/tpspace/ptgCCS.go

```diff
@@ -0,0 +1,59 @@
+package tpspace
+
+import (
+	"math"
+
+	rutils "go.viam.com/rdk/utils"
+)
+
+// This does something with circles
+// Other ptgs will be based on this ptg somehow.
```

Comment:
They do for now.

This may change in the future, and I'd prefer to keep things explicit for now & for easier tracing

Note:
https://github.com/uber-go/guide/blob/master/style.md#avoid-embedding-types-in-public-structs